atom/browser/api/atom_api_auto_updater.cc

```diff
@@ -62,16 +62,20 @@ void AutoUpdater::OnError(const std::string& message,
   v8::Locker locker(isolate());
   v8::HandleScope handle_scope(isolate());
   auto error = v8::Exception::Error(mate::StringToV8(isolate(), message));
-  auto errorObject = error->ToObject(
-    isolate()->GetCurrentContext()).ToLocalChecked();
```

Comment:
Should `AutoUpdater::OnError(const std::string& message)` also be changed to emit `error` rather than `error->ToObject(...)`?